src/geotop/math.optim.h

```diff
@@ -1,17 +1,20 @@
 #ifndef __math_optim_h_
 #define __math_optim_h_
 
+#include "config.h"
+
 #include <cmath>
+#include <fstream>
```

Comment:
why you need this header here?

Reply:
it was useful to print the bases and the exponents to a file but now it has to be removed